File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    s=s.lower()
    s=s.title()
    answer=''
    answer+=s[0]
    for i in range(len(s)-1):
        if s[i].isdigit():
            answer+=s[i+1].lower()
        else:
            answer+=s[i+1]
    return answer
        

    

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        StringBuilder answer = new StringBuilder();
        Boolean firstletter =true;
        char[] arr =s.toCharArray();
        for(char voca : arr){
            if(voca!=' '&&firstletter){
                String a =""+voca;
                answer.append(a.toUpperCase());
                firstletter = false;
            }else if(voca!=' '&&!firstletter){
                String a =""+voca;
                answer.append(a.toLowerCase());
            }else if(voca==' '){
                answer.append(" ");
                firstletter=true;
            }
        }
        
        
        return answer.toString();
    }
}